homework-service/HWS/HWS.Web/Services/GroupService.cs

```diff
@@ -41,6 +41,11 @@ public async Task<Group> GetGroup(Guid id)
             return await groupRepository.FindById(id);
         }
 
+        public async Task<Comment> CreateGroupComment(User user, Group group, string content)
```

Comment:
Itt érdemes létrehozni a Domain.Comment-et és beállítani a fontosabb adatokat. Kb, amit a repository-ban csináltál.